### ISSUE TITLE
Bounce : important information missing

### ISSUE THREAD

**Issue by chandon** [mautic/documentation#80]
For bounce detection, mautic uses Return-path to detect wich message originated the bounce. For example, if you send an email from **my.address@email.com**, mautic will insert a Return-path: **my.address+bounce_65456489798@email.com**

So if your mailbox isn't configured to accept ALL email with +bounce... the mailbox will never be able to receive the bounce. And, of course, the detection will not work

So you HAVE TO activate this option with your mailbox if you want receive bounces with email sent from mautic (and it's not a standard option...)

There is no mention in the documentation

**Comment by escopecz** [comment]
Good point! Could you add this not to the documentation? https://github.com/mautic/documentation#how-to-contribute-to-the-docs
